### PR TITLE
Add extra income / explanation page

### PR DESCRIPTION
**Pull request by jessieay**
screen shot 2017-08-14 at 4 53 40 pm

screen shot 2017-08-14 at 4 53 35 pm

### CODE REVIEW

**Comment by jessieay** [comment]
[As an MDHHS worker, I want to know details about the income and employment of each HH member to help me determine eligibility](https://trello.com/c/Vrj3OSyh/107-as-an-mdhhs-worker-i-want-to-know-details-about-the-income-and-employment-of-each-hh-member-to-help-me-determine-eligibility)

File: app/helpers/mb_form_builder.rb [hunk 1]
```diff
@@ -20,6 +20,24 @@ def mb_input_field(
...
+    notes: [],
+    options: {},
+    classes: [],
+    placeholder: nil,
+    autofocus: nil
+  )
+    classes = classes.append(%w[textarea])
+    <<-HTML.html_safe
+      <fieldset class="form-group#{error_state(object, method)}">
+        #{label_and_field(method, label_text, text_area(method, { autofocus: autofocus, class: classes.join(' '), autocomplete: 'off', autocorrect: 'off', autocapitalize: 'off', spellcheck: 'false', placeholder: placeholder }.merge(options)), notes: notes)}
```
Comment (jayroh) [inline app/helpers/mb_form_builder.rb]: possible to break this line up into several local vars? I'm surprised rubocop didn't complain about this one

File: app/steps/step_navigation.rb [hunk 2]
```diff
@@ -13,6 +13,10 @@ class StepNavigation
       PersonalDetailController,
       HouseholdMembersOverviewController,
     ],
+    "Money & Income" => [
+      IncomeChangeController,
+      IncomeChangeExplanationController,
+    ],
```
Comment (jayroh) [inline app/steps/step_navigation.rb]: The number of steps in this app are growing like weeds! :seedling: 

Good weeds!

File: spec/features/snap_app_spec.rb [hunk 3]
```diff
@@ -40,7 +40,29 @@
...
+
+    # this is next
+    # on_page "Money & Income" do
+    #   within(find(".household-member-group", text: "Jessie (that's you!)")) do
+    #     choose("Employed")
+    #   end
+    #   within(find(".household-member-group", text: "Joey")) do
+    #     choose("Not Employed")
+    #   end
+    # end
```
Comment (jayroh) [inline spec/features/snap_app_spec.rb]: Delete this comment block?

Comment (jessieay) [inline spec/features/snap_app_spec.rb]: sure -- was just saving for future me :) (that's the next thing I am working on)

Comment (jessieay) [inline spec/features/snap_app_spec.rb]: removed at  0a7ffbd

File: app/helpers/mb_form_builder.rb (hunk 1 above)
Comment (jessieay) [inline app/helpers/mb_form_builder.rb]: yeah you are right, weird that it didnt correct. Copied this over from GCF.

Anyway, cleaned up at 5371e9e

**Pull request merged**